cypress/e2e/signIn.cy.js

```
@@ -1,7 +1,32 @@
 /// <reference types="cypress" />
 
+const { faker } = require('@faker-js/faker');
+
 describe('Sign In page', () => {
+  const username = `${faker.person.firstName()}_${faker.person.lastName()}`;
+  const email = faker.internet.email().toLowerCase();
+  const password = faker.internet.password();
+
+  before(() => {
+    cy.request({
+      method: 'POST',
+      url: '/api/users',
+      body: { user: { username, email, password } },
+      failOnStatusCode: false
+    });
```

The `cy.request` block creates the user which is correct, but `failOnStatusCode: false` will hide failures from the API and let the test continue when account creation failed. Remove `failOnStatusCode: false` or assert the response explicitly (for example `cy.request(...).then(resp => expect(resp.status).to.eq(200))`) so the test fails fast if the user wasn't created.